libs/java/server_common/src/test/java/com/yahoo/athenz/common/server/util/config/providers/ConfigProviderAwsParametersStoreTest.java

```diff
@@ -0,0 +1,80 @@
+package com.yahoo.athenz.common.server.util.config.providers;
```

Comment:
please add the copyright notice with Athenz Authors

Reply:
Fixed